### PR TITLE
fix phases display on homepage

### PR DESCRIPTION
**Pull request by pascalhuynh**
Everything is said in the title

### CODE REVIEW

File: assembl/static2/js/app/components/home/phases/phase.jsx [hunk 1]
```diff
@@ -38,20 +59,33 @@ class Phase extends React.Component {
...
+    const { imgUrl, startDate, title, description, index, screenWidth } = this.props;
     const stepNumber = index + 1;
+    const backgroundImage = imgUrl ? `url(${imgUrl})` : null;
+
+    // Return top margin value according to the screenwidth
+    const boxTopMargin = () => {
+      if (screenWidth < SM_SCREEN_WIDTH) {
+        const { maxTopMargin, marginStep, phasesPerLine } = BOX_TOP_MARGIN.mobile;
+        return maxTopMargin - (index % phasesPerLine) * marginStep;
+      } else if (screenWidth >= SM_SCREEN_WIDTH && screenWidth < LG_SCREEN_WIDTH) {
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/home/phases/phase.jsx]: These conditions should be functions, like `isMidsizedScreen` or something of that nature. There is also the `isMobile()`, perhaps can add these functions to the prototype of this object.

File: assembl/static2/css/views/home.scss [hunk 2]
```diff
@@ -199,45 +201,62 @@
...
+
+      &:nth-child(6) {
+        margin-top: 30px;
+      }
+
+      &:nth-child(7) {
+        margin-top: 20px;
+      }
+
+      &:nth-child(8) {
```
[the remark this answers is not in the document]
Comment (pascalhuynh) [reply assembl/static2/css/views/home.scss]: Just like you said: this is a quick fix this component should definitely be restructured just like how I did on this fix: it took me a lot of time to removed a bunch of unneeded code and patch transversal CSS dependencies (just like some other BH tickets btw). We made a clear choice when we updated the UI: quick fix, quick patch and only restructure what really need to be restructured

File: assembl/static2/js/app/components/home/phases/phase.jsx (hunk 1 above)
Comment (pascalhuynh) [reply assembl/static2/js/app/components/home/phases/phase.jsx]: updated !

**Comment by pascalhuynh** [comment]
code were updated according to the comment. Will merge when Travis verification is done.

**Pull request merged**